platform/darwin/src/MGLAccountManager.m

```diff
@@ -28,6 +30,13 @@ + (void)load {
     if (accessToken.length) {
         self.accessToken = accessToken;
     }
+    
+    NSString *apiBaseURL = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"MGLMapboxAPIBaseURL"];
+    
+    // If apiBaseURL is not a valid URL, [NSURL URLWithString:] will be `nil`.
+    if (apiBaseURL.length && [NSURL URLWithString:apiBaseURL]) {
+        [self setAPIBaseURL:[NSURL URLWithString:apiBaseURL]];
```

Comment:
Rather than initializing two (hopefully) identical URL objects, do it once as a variable.